src/ontology/mondo-edit.obo

```diff
@@ -562163,7 +562200,7 @@ synonym: "TFAP2B-related PDA and Char syndrome spectrum disorder" EXACT CLINGEN_
 is_a: MONDO:0005453 {source="https://clinicalgenome.org/affiliation/40060/"} ! congenital heart disease
 intersection_of: MONDO:0005453 ! congenital heart disease
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/11743 ! TFAP2B
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/11743 {source="OMIM:169100", source="OMIM:617035"} ! TFAP2B
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/11743 ! TFAP2B
```

Comment:
these should have also been maintained `relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/11743 {source="OMIM:169100", source="OMIM:617035"} ! TFAP2B` based on the [PR](https://github.com/monarch-initiative/mondo/pull/8637/files#diff-889e0a9ba62d62cc6ee7c5c9bead0075444d506253590ef0dcf4786904ac6768R564718) where this Mondo term was created